packages/backend/src/index.ts

```diff
@@ -40,10 +30,19 @@ backend.add(
         'configSchema.json',
       );
     },
+    moduleLoader: logger => new CommonJSModuleLoader(logger),
+    logger: config => {
+      const auditLogConfig = config?.getOptionalConfig('auditLog');
+      return {
+        transports: [
```

Comment:
Didn't you miss the [audit log specific format](https://github.com/janus-idp/backstage-showcase/pull/1818/files#diff-77fa0019c0266a3c6ce82e398cc562233f1cbeec51c7dbd299724ee7c800212aL106) here ?

Reply:
It's not necessary.

Reply:
why ?

Reply:
`format` is the fallback is it `transports` doesn't work.